README.md

```diff
@@ -32,6 +32,15 @@ To install jazzy, run `[sudo] gem install jazzy` from your command line.
 Run `jazzy` from your command line. Run `jazzy -h` for a list of additional
 options.
 
+### Troubleshooting
+
+***Only extensions are listed in the documentation.***
+By default, jazzy will only create documentation for public methods and classes. If you want to generate documentation for all of your files, make sure you set the `--min-acl` flag to `internal` or `private`.
+
+```sh
+jazzy --output /path/to/output --source-directory /path/to/src --min-acl internal
```

Comment:
I think the paragraph above is sufficient to describe how to use the `--min-acl` parameter. I'd remove this sample usage snippet.